server/app/graphql/mutations/delete_user_artist_relationship.rb

```diff
@@ -19,7 +19,7 @@ def resolve(user_name:, artist_name:)
       user = User.find_by(name: user_name)
       artist = Artist.find_by(name: artist_name)
       user_artist_relationship = UserArtist.find_by(user: user, artist: artist)
-      if !user_artist_relationship.destroy
+      if !user_artist_relationship || !user_artist_relationship.destroy
```

Comment:
`user_artist_relationship.destroy` が失敗した時に""Not following"を返すのは違和感を感じた。
今回はそんなに拘らなくて良さそうやから，"Error"とかの文字列返しておけばいいかな？
別にifの条件分岐増やしても良いと思う

Reply:
追記
そもそも，`user_artist_relationship.destroy`が失敗するケースがほとんどないだろうから，
```suggestion
      if !user_artist_relationship
        ~~
      else
         user_artist_relationship.destroy!
      end
```

とかでいい気がした